circle.yml

```diff
@@ -1,24 +0,0 @@
-machine:
```

Comment:
Why are you removing Circle?

Reply:
I am NOT.

According to Cricle CI 2.0 migration notes, it is best to have this in `.circleci/config.yml`, hence moved it.

See - https://github.com/dequelabs/axe-matchers/pull/42/files#diff-1d37e48f9ceff6d8030570cd36286a61

And the respective CI - https://circleci.com/gh/dequelabs/axe-matchers/102